auto_tests/group_message_test.c

```diff
@@ -27,7 +27,7 @@ static void group_invite_handler(Tox *tox, uint32_t friend_number, const uint8_t
     self_info.nick_length = 4;
     self_info.user_status = TOX_USER_STATUS_NONE;
 
-    printf("invite arrived; accepting\n");
+    printf("invite arrived; accepting; tox0_index=%u, tox1_index=%u\n", global_state_tox0->index, global_state_tox1->index);
```

Comment:
`error: ‘global_state_tox0’ undeclared (first use in this function)`

It'll need to move up. But also, let's not do this in this PR, because what we actually want is no use of global state in this test. If you do that, you'll notice it's quite a bit of work.